### ISSUE TITLE
Multi arch images

### ISSUE THREAD

**Issue by AlexisColes** [mlocati/docker-php-extension-installer#1005]
Although it seems to work fine using this image on arm64, we do get a warning in our pipelines which would be nice to cleanup.

![image](https://github.com/user-attachments/assets/56292545-af4b-4553-b93a-02c3e762776a)

Could you publish this image behind a manifest that support different arches please.

**Comment by mlocati** [comment]
> Could you publish this image behind a manifest that support different arches please.

Pull requests are welcome

**Comment by baschny** [comment]
@mlocati how do you upload the docker images to docker hub? This workflow doesn't seem to be part of this repo.

In case you are using the "Automatic Build" functionality of "Docker Hub" (i.e. have the hooks installed here on Github to build them on Docker Hub on releases) there is (AFAIK) **no way** of having it create multi-arch images.

If you are doing it manually from your local machine, you can use buildx to create a multiarch push of "latest":
```
docker buildx create --name multiarch --use --bootstrap
docker buildx build --push \
	--platform linux/amd64,linux/arm64 \
	--tag mlocati/php-extension-installer:latest .
docker buildx rm multiarch
```

Since you are using Github Actions already for testing the image, it would be "easy" to add a github action which pushes to Docker Hub on release (instead of having Docker Hub pulling and "automatic building"). There are ready made actions to help with this setup (`docker/setup-buildx-action@v3`, `docker/login-action@v3`, `docker/build-push-action@v6` etc).

If you want to go this route, i could provide you with an pull request with such an github action.

**Comment by baschny** [comment]
This "warning" appears seems to be new in later docker versions (appears on console when building on ARM - Apple Silicon), and also on Github Actions - that's why its probably popping up only now.

**Comment by mlocati** [comment]
> how do you upload the docker images to docker hub? This workflow doesn't seem to be part of this repo.

See
https://github.com/mlocati/docker-php-extension-installer/blob/2.6.4/.github/workflows/readme-release.yml#L173-L181

**Comment by baschny** [comment]
Cool! I have missed that GA. My additions to make it build for arm also: #1009

**Comment by mlocati** [comment]
I've just published version 2.7.0 with multi-arch support (thanks @baschny!)
